### DISCUSSION TITLE
[Question] - can I recopy monikas chr file onto google drive or a flash drive

### DISCUSSION THREAD

**Discussion by JinSakaiM** [Monika-After-Story/MonikaModDev#8737] [QA - tech support + general] [answered]
this is basically a quick question but can I recopy Monikas chr file onto google drive or a flash drive?

**Comment by Booplicate** [comment, marked as answer]
Just like any other files, you can probably copy this one. Bear in mind, `monika.chr` is useless in MAS.

**Comment by JinSakaiM** [comment]
so if I deleted it it would be useless? 👀

**Reply by Booplicate** [reply]
You can delete it or keep, MAS does not use `.chr` files, Monika deletes them at the start. Her file is named literally `monika`. That one is important since it's her.
